kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnector.java

```diff
@@ -86,4 +89,15 @@ public ConfigDef config() {
     return S3SinkConnectorConfig.getConfig();
   }
 
+  @Override
+  public Config validate(final Map<String, String> connectorConfigs) {
+    Config config = super.validate(connectorConfigs);
+    // Checking whether the bucket exists
+    if (!S3BucketCheck.checkBucketExists(new S3SinkConnectorConfig(connectorConfigs))) {
+      throw new ConnectException("Bucket does not exist or invalid credentials "
+              + "provided");
```

Comment:
We don't want to throw an exception, because doing so essentially means the validation check could not do its thing. But what we want is for a validation messages to be captured on the bucket property in the `Config` response.